views/modal/wp_editor_link.php

```diff
@@ -0,0 +1,62 @@
+<div id="wp-link-backdrop" style="display: none"></div>
```

Comment:
Add this file comment

```
<?php
/**
 * WP Editor link modal
 *
 * @package Tutor\Views
 * @author Themeum <support@themeum.com>
 * @link https://themeum.com
 * @since 3.0.0
 */
```

Reply:
Added